### PR TITLE
Complete the missing translations

### PR DESCRIPTION
**Pull request by fricyo**

### CODE REVIEW

**Comment by FooIbar** [comment]
Thank you!

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -453,14 +467,22 @@
     <string name="settings_privacy">隱私</string>
     <string name="settings_privacy_secure">不允許螢幕擷取</string>
     <string name="settings_privacy_secure_summary">啟用後，將無法對本程式進行螢幕擷取，同時，將不會在系統任務切換器中顯示該程式的預覽內容，重新啟動本程式以套用此變更</string>
+    <string name="clear_search_history">清除裝置上的搜尋記錄</string>
+    <string name="clear_search_history_summary">移除曾在此裝置上進行過的搜尋</string>
+    <string name="clear_search_history_confirm">清除搜尋記錄？</string>
+    <string name="search_history_cleared">已清除搜尋記錄</string>
     <string name="download_service">下載服務</string>
     <string name="favorite_name">收藏圖庫</string>
+    <string name="dark_theme">深色主題</string>
```
Comment (FooIbar) [inline app/src/main/res/values-zh-rTW/strings.xml:476]: Dark theme is already translated here. https://github.com/FooIbar/EhViewer/blob/e2f024ae2917c11198172bb58e63782b82e3dfd2/app/src/main/res/values-zh-rTW/strings.xml#L596

**Review by FooIbar — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-zh-rTW/strings.xml (hunk 1 above)
Comment (fricyo) [reply app/src/main/res/values-zh-rTW/strings.xml:476]: Sorry, my oversight. What should I do now? I'm not familiar with this

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 2]
```diff
@@ -320,6 +327,8 @@
     <string name="settings_eh_thumb_resolution">縮圖解析度</string>
     <string name="settings_eh_thumb_resolution_summary">%s，指定解析度可能會使縮圖取得失敗</string>
     <string name="settings_eh_thumb_resolution_auto">自動</string>
+    <string name="settings_eh_force_eh_thumb">強制 e-hentai 縮圖/API 伺服器</string>
+    <string name="settings_eh_force_eh_thumb_summary">若exhentai載入遇到問題可嘗試此項</string>
```
Comment (FooIbar) [inline app/src/main/res/values-zh-rTW/strings.xml:331]: I just updated this string, you may want to check it https://github.com/FooIbar/EhViewer/commit/256dedcb7cec886fe9c4bddcda0098f5f8e37a13

**Review by FooIbar — CHANGES_REQUESTED** [review]

**Comment by FooIbar** [comment]
LGTM, thank you for your contribution!

**Pull request merged**